src/styles/card.scss

```diff
@@ -0,0 +1,71 @@
+@import 'button';
+@import 'stars';
+
+.card {
+  width: 200px;
+  border-radius: 5px;
+  background-color: $white;
+  padding: 16px;
+  border: 1px solid $milky;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    background-image: url('../images/imac.jpeg');
+    background-size: 100% 100%;
+    margin: 16px auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin-bottom: 4px;
+    color: $black;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
```

Comment:
Default value, so you can just remove it everywhere
```suggestion
```

Reply:
Yes, it was my mistake and it was used only there. Removed now. Thanks.